share/lib/api_call.sh

```diff
@@ -66,6 +68,24 @@ full_api_call() {
     echo "A token is mandatory to query the server"
     exit 1
   fi
+
   eval ${DOWNLOAD_COMMAND} ${HEADER_OPT} "\"X-API-Token: ${token}\"" ${HEADER_OPT} "\"Content-Type: application/json;charset=utf-8\"" ${HEADER_OPT} "\"X-API-Version: latest\"" \"${url}${api}\"
+
+}
+
+# This function make an API call
+complete_api_call() {
```

Comment:
This name is meaningless
There should be an explanation of why it is different from the precedent function